netfox/Core/NFXProtocol.swift

```diff
@@ -61,7 +61,8 @@ open class NFXProtocol: URLProtocol
         req = (NFXProtocol.canonicalRequest(for: request) as NSURLRequest).mutableCopy() as! NSMutableURLRequest
         
         self.model?.saveRequest(req as URLRequest)
-                
+//        NFXPathNodeManager.sharedInstance.add(model!)
+        
```

Comment:
Either remove commented code (I'm guessing it was just some test code), or add a `//TODO` with a description.